### ISSUE TITLE
Crash, log included.

### ISSUE THREAD

**Issue by doronbehar** [syncthing/syncthing#4549]
### Version Information

Syncthing Version: v0.14.40
OS Version: Android 7.1.2 - Lineage OS 14.1 nightly

[syncthing.log](https://gist.github.com/doronbehar/be7c213612a860ae039c2587f87007ba)

**Comment by calmh** [comment]
The log unfortunately lacks any information on where the crash occurred; I’m not sure if it’s just not complete or how it was written. What did you see when this happened?

**Comment by doronbehar** [comment]
First of all, I'm sorry but I rushed when I reported the issue, it occurred to me on my android device (Edited the first comment) and I just copied the log to my PC in order to report the issue. Should it have been reported in [syncthing-android](https://github.com/syncthing/syncthing-android)?

I think what could have made the error appear is that I launched for the 1st time [sieren/QSyncthingTray](https://github.com/sieren/QSyncthingTray). I'm not totally sure about it but I don't think the crash will appear again.

**Comment by calmh** [comment]
I don't know if you mean to say you run qsyncthingtray on your phone or if you think running that caused a crash on another device... But without a more complete crash log there is not much we can do.

**Comment by doronbehar** [comment]
Well, it might sound pretty bizarre and I'm not totally sure about it but I think that the crash happened when I connected `qsyncthingtray`. To be clear, **It happened on my Android Device**.

I wasn't harmed in anyway by this crash, I just restarted the app and everything worked fine from then on. I just thought that the right thing to do is to report it. It's defiantly not reproducible and if you guys can't gain benefit about potential harmful bugs about, I won't mind you close it.
